test/runtests.jl

```diff
@@ -47,18 +48,23 @@ StableHashTraits.hash_method(::TestType, ::MyContext) = UseQualifiedName(UseProp
     @test stable_hash([1, 2, 3]) == 0x1a366aea
     @test stable_hash((a=1, b=2)) == 0x240bb84c
     @test stable_hash(sin) == 0x7706a39f
-    @test stable_hash(TestType2(1, 2)) == 0x1f99ed3b
+    @test stable_hash(TestType2(1, 2)) == 0x8c28b28a
```

Comment:
this is a breaking change to the package, btw

Reply:
Ah yes, I will at least mark it as such.

Do you think that is sufficient? I think with a little more work and a few extra methods specific to `crc32c` I could probably make sure these remain unchanged.